ext/liquid_c/vm_assembler.c

```diff
@@ -36,8 +36,12 @@ void vm_assembler_gc_mark(vm_assembler_t *code)
                 break;
```

Comment:
Now that the constants buffer just has ruby constants in it, we can mark it as simply as we do the stack, by just using `rb_gc_mark_locations((VALUE *)vm->constants.data, (VALUE *)vm->constants.data_end);`